scalopus_tracing/src/native/tracepoint_collector_native.h

```diff
@@ -158,6 +151,43 @@ class TracePointCollectorNative : public MapTracker<unsigned long, tracepoint_co
    * @brief Set the size of any new ringbuffers that will be created.
    */
   void setRingbufferSize(std::size_t size);
+
+  /**
+   * @brief Return the map of active threads and their ringbuffers
+   */
+  BufferMap::MapType getActiveMap() const;
+
+  /**
+   * @brief Retrieve the orphaned buffers and clear them.
+   */
+  BufferVector retrieveOrphanedBuffers();
```

Comment:
Maybe rename to `retrieveAndClearOrphanedBuffers` :thinking:

Reply:
Yeah, I guess that's more clear, changed it :+1: